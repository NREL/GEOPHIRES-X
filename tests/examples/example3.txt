GEOPHIRES v2.0 Input File
Geothermal Combined Heat and Power Problem using a Thermal Drawdown Reservoir Model and BICYCLE Economic Model

Example 3 description: This example problem considers an EGS reservoir at 3.1km depth.
The heat is used in a combined heat and power topping cycle model with double flash as
topping cycle and electricity considered as the main product.

***Subsurface technical parameters***
*************************************
Reservoir Model,3,						---m/A Single Fracture Thermal Drawdown
Drawdown Parameter,.00002,					---[kg/s/m2]
Reservoir Depth,3.1,						---[km]
Number of Segments,1,						---[-]
Gradient 1,70,							---[deg.C/km]
Maximum Temperature,400,					---[deg.C]
Number of Production Wells,3,					---[-]
Number of Injection Wells,3,					---[-]
Production Well Diameter,8.5,					---[inch]
Injection Well Diameter,8.5,					---[inch]
Ramey Production Wellbore Model,0,				---Should be 0 (disabled) or 1 (enabled)
Production Wellbore Temperature Drop,5,				---[deg.C]
Injection Wellbore Temperature Gain,3,				---[deg.C]
Production Flow Rate per Well,70,				---[kg/s]
Reservoir Volume Option,1,					---Should be 1 2 3 or 4. See manual for details.
Fracture Shape,1,						---Should be 1 2 3 or 4
Fracture Area,200000,						---[m2]
Number of Fractures,12,						---[-]
Fracture Separation,80,						---[m]
Injectivity Index,5,						---[kg/s/bar]
Injection Temperature,70,					---[deg.C]
Maximum Drawdown,1,						---[-] no redrilling considered
Reservoir Heat Capacity,1000,					---[J/kg/K]
Reservoir Density,3000,						---[kg/m3]
Reservoir Thermal Conductivity,3,				---[W/m/K]
Water Loss Fraction,0.02,						---[-]

***Surface Technical Parameters***
**********************************

End-Use Option,31,						---CHP Topping Cycle with electricity as the main product
Power Plant Type,4,						---Double-Flash
Circulation Pump Efficiency,.80,				---[-]
Utilization Factor,.9,						---[-]
End-Use Efficiency Factor,.9,					---[-]
Surface Temperature,15,						---[deg.C]
Ambient Temperature,15,						---[deg.C]

***Financial Parameters***
**************************

Plant Lifetime,35,						---[years]
Economic Model,3,						---BICYCLE Levelized Cost Model
Fraction of Investment in Bonds,.5,				---[-] Required for BICYCLE model
Inflated Bond Interest Rate,.05,				---[-] Required for BICYCLE model
Inflated Equity Interest Rate,.08,				---[-] Required for BICYCLE model
Inflation Rate,.02,						---[-] Required for BICYCLE model
Combined Income Tax Rate,.3,					---[-] Required for BICYCLE model
Gross Revenue Tax Rate,0,					---[-] Required for BICYCLE model
Investment Tax Credit Rate,0,					---[-] Required for BICYCLE model
Property Tax Rate,0,						---[-] Required for BICYCLE model
Inflation Rate During Construction,0.05,				---[-]

***Capital and O&M Cost Parameters***
*************************************

Well Drilling and Completion Capital Cost Adjustment Factor,1, 	---[-] use built in correlations
Well Drilling Cost Correlation,1,				---[-] use built in correlations
Reservoir Stimulation Capital Cost Adjustment Factor,1,		---[-] use built in correlations
Surface Plant Capital Cost Adjustment Factor,1,			---[-] use built in correlations
Field Gathering System Capital Cost Adjustment Factor,1,	---[-] use built in correlations
Exploration Capital Cost Adjustment Factor,1,			---[-] use built in correlations
Wellfield O&M Cost Adjustment Factor,1,				---[-] use built in correlations
Surface Plant O&M Cost Adjustment Factor,1,			---[-] use built in correlations
Water Cost Adjustment Factor,1,					---[-] use built in correlations
Heat Rate,.02,							---[$/kWh]

***Simulation Parameters***
***************************

Print Output to Console,1,					---Should be 1 (to print) or 0 (to not print)
Time steps per year,10,						---[-]
