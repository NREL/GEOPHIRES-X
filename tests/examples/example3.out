                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.3.3
 GEOPHIRES Build Date: 2022-06-30
 Simulation Date: 2024-02-01
 Simulation Time:  11:25
 Calculation Time:      0.107 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Cogeneration Topping Cycle, Heat sales considered as extra income
      Average Net Electricity Production:                    19.38 MW
      Average Direct-Use Heat Production:                    10.86 MW
      Electricity breakeven price:                            5.79 cents/kWh
      Number of production wells:                             3
      Number of injection wells:                              3
      Flowrate per production well:                          70.0 kg/sec
      Well depth (or total length, if not vertical):          3.1 kilometer
      Geothermal gradient:                                    0.0700 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model  = BICYCLE
      Accrued financing during construction:                  5.00
      Project lifetime:                                      35 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                          -3.35 MUSD
      Project IRR:                                           0.06 %
      Project VIR=PI=PIR:                                    0.97
      Project MOIC:                                          0.63

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             3
      Number of Injection Wells:                              3
      Well depth (or total length, if not vertical):          3.1 kilometer
      Water loss rate:                                        2.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 73.0 degC
      User-provided production well temperature drop
      Constant production well temperature drop:              5.0 degC
      Flowrate per production well:                          70.0 kg/sec
      Injection well casing ID:                               8.500 in
      Production well casing ID:                              8.500 in
      Number of times redrilling:                             0
      Power plant type:                                       Double-Flash


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0700 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Single Fracture m/A Thermal Drawdown Model
      m/A Drawdown Parameter:                                 0.00002 1/year
      Bottom-hole temperature:                              232.00 degC
      Reservoir volume calculated with fracture separation and number of fractures as input
      Number of fractures:                                   12.00
      Fracture separation:                                   80.00 meter
      Reservoir volume:                               176000000 m**3
      Reservoir hydrostatic pressure:                       100.00 kPa
      Plant outlet pressure:                                100.00 kPa
      Injectivity Index:                                      5.00 kg/sec/bar
      Reservoir density:                                   3000.00 kg/m**3
      Reservoir thermal conductivity:                         3.00 watt/m/kelvin
      Reservoir heat capacity:                             1000.00 J/kg/kelvin


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       227.0 degC
      Average Production Temperature:                       219.5 degC
      Minimum Production Temperature:                       205.2 degC
      Initial Production Temperature:                       227.0 degC
      Average Reservoir Heat Extraction:                    132.05 MW
      Wellbore Heat Transmission Model = Constant Temperature Drop:       5.0 degC
      Average Injection Well Pump Pressure Drop:            742.5 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      34.45 MUSD
         Drilling and completion costs per well:              5.74 MUSD
         Stimulation costs:                                   4.53 MUSD
         Surface power plant costs:                          64.13 MUSD
         Field gathering system costs:                        3.18 MUSD
         Total surface equipment costs:                      67.31 MUSD
         Exploration costs:                                   5.51 MUSD
      Total capital costs:                                  111.79 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.83 MUSD/yr
         Power plant maintenance costs:                       2.33 MUSD/yr
         Water costs:                                         0.11 MUSD/yr
      Total operating and maintenance costs:                  3.27 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.23 MW/(kg/s)
      Maximum Total Electricity Generation:                  21.16 MW
      Average Total Electricity Generation:                  19.58 MW
      Minimum Total Electricity Generation:                  16.69 MW
      Initial Total Electricity Generation:                  21.16 MW
      Maximum Net Electricity Generation:                    20.96 MW
      Average Net Electricity Generation:                    19.38 MW
      Minimum Net Electricity Generation:                    16.49 MW
      Initial Net Electricity Generation:                    20.96 MW
      Average Annual Total Electricity Generation:          153.96 GWh
      Average Annual Net Electricity Generation:            152.36 GWh
      Initial pumping power/net installed power:            100.00 %
      Maximum Net Heat Production:                           11.90 MW
      Average Net Heat Production:                           10.86 MW
      Minimum Net Heat Production:                            8.78 MW
      Initial Net Heat Production:                           11.90 MW
      Average Annual Heat Production:                        85.40 GWh
      Average Pumping Power:                                  0.20 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR     THERMAL             GEOFLUID             PUMP             NET              NET             FIRST LAW
           DRAWDOWN           TEMPERATURE           POWER           POWER             HEAT            EFFICIENCY
                                (deg C)             (MW)            (MW)              (MW)               (%)
   0         1.0000              227.00             0.2037         20.9570             11.9036              16.6854
   1         1.0000              227.00             0.2037         20.9570             11.9036              16.6854
   2         1.0000              227.00             0.2037         20.9570             11.9036              16.6854
   3         1.0000              227.00             0.2037         20.9570             11.9036              16.6854
   4         1.0000              227.00             0.2037         20.9567             11.9034              16.6853
   5         0.9999              226.99             0.2037         20.9541             11.9019              16.6844
   6         0.9998              226.95             0.2037         20.9454             11.8967              16.6815
   7         0.9994              226.86             0.2037         20.9258             11.8851              16.6749
   8         0.9987              226.70             0.2037         20.8911             11.8645              16.6631
   9         0.9976              226.46             0.2037         20.8386             11.8333              16.6453
  10         0.9962              226.13             0.2037         20.7671             11.7905              16.6211
  11         0.9944              225.72             0.2037         20.6768             11.7361              16.5904
  12         0.9922              225.22             0.2037         20.5686             11.6705              16.5535
  13         0.9896              224.65             0.2037         20.4442             11.5945              16.5109
  14         0.9868              224.00             0.2037         20.3053             11.5088              16.4632
  15         0.9837              223.30             0.2037         20.1539             11.4144              16.4110
  16         0.9804              222.54             0.2037         19.9921             11.3123              16.3548
  17         0.9768              221.74             0.2037         19.8215             11.2034              16.2954
  18         0.9731              220.90             0.2037         19.6439             11.0886              16.2332
  19         0.9693              220.03             0.2037         19.4607             10.9687              16.1687
  20         0.9654              219.14             0.2037         19.2734             10.8444              16.1024
  21         0.9614              218.23             0.2037         19.0831             10.7163              16.0346
  22         0.9573              217.30             0.2037         18.8907             10.5851              15.9656
  23         0.9531              216.36             0.2037         18.6973             10.4512              15.8959
  24         0.9490              215.41             0.2037         18.5035             10.3152              15.8256
  25         0.9448              214.46             0.2037         18.3100             10.1774              15.7549
  26         0.9406              213.51             0.2037         18.1173             10.0383              15.6841
  27         0.9364              212.56             0.2037         17.9259              9.8981              15.6133
  28         0.9322              211.61             0.2037         17.7362              9.7571              15.5427
  29         0.9280              210.66             0.2037         17.5484              9.6156              15.4724
  30         0.9239              209.73             0.2037         17.3628              9.4738              15.4024
  31         0.9198              208.79             0.2037         17.1797              9.3320              15.3330
  32         0.9157              207.87             0.2037         16.9991              9.1902              15.2640
  33         0.9117              206.95             0.2037         16.8213              9.0486              15.1957
  34         0.9077              206.04             0.2037         16.6463              8.9075              15.1280


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             HEAT                 ELECTRICITY                HEAT              RESERVOIR        PERCENTAGE OF
                  PROVIDED               PROVIDED                EXTRACTED          HEAT CONTENT    TOTAL HEAT MINED
                 (GWh/year)             (GWh/year)               (GWh/year)          (10^15 J)           (%)
   1                93.8                  165.2                   1094.51               80.01                 4.69
   2                93.8                  165.2                   1094.51               76.07                 9.39
   3                93.8                  165.2                   1094.51               72.13                14.08
   4                93.8                  165.2                   1094.51               68.19                18.77
   5                93.8                  165.2                   1094.47               64.25                23.47
   6                93.8                  165.2                   1094.30               60.31                28.16
   7                93.8                  165.1                   1093.85               56.37                32.85
   8                93.6                  164.9                   1092.98               52.44                37.54
   9                93.4                  164.5                   1091.57               48.51                42.22
  10                93.1                  164.0                   1089.56               44.59                46.89
  11                92.7                  163.4                   1086.92               40.67                51.55
  12                92.3                  162.6                   1083.68               36.77                56.20
  13                91.7                  161.7                   1079.87               32.89                60.83
  14                91.1                  160.6                   1075.53               29.01                65.44
  15                90.4                  159.5                   1070.74               25.16                70.03
  16                89.6                  158.3                   1065.55               21.32                74.60
  17                88.8                  157.0                   1060.01               17.51                79.15
  18                87.9                  155.6                   1054.18               13.71                83.67
  19                87.0                  154.2                   1048.11                9.94                88.16
  20                86.0                  152.7                   1041.84                6.19                92.63
  21                85.0                  151.2                   1035.42                2.46                97.07
  22                84.0                  149.7                   1028.87               -1.24               101.48
  23                82.9                  148.2                   1022.24               -4.92               105.87
  24                81.9                  146.6                   1015.54               -8.58               110.22
  25                80.8                  145.1                   1008.79              -12.21               114.55
  26                79.7                  143.6                   1002.03              -15.82               118.84
  27                78.6                  142.1                    995.26              -19.40               123.11
  28                77.5                  140.6                    988.51              -22.96               127.35
  29                76.4                  139.1                    981.77              -26.49               131.56
  30                75.3                  137.6                    975.07              -30.01               135.74
  31                74.1                  136.2                    968.42              -33.49               139.89
  32                73.0                  134.7                    961.81              -36.95               144.02
  33                71.9                  133.3                    955.26              -40.39               148.11
  34                70.8                  131.9                    948.78              -43.81               152.18
  35                62.8                  117.6                    848.41              -46.86               155.82
